bayesian/utils.py

```diff
@@ -369,9 +378,7 @@ def get_analyses_from_graph(ecosystem, package, version):
                         "recommended_versions": result_data[0]['package']['latest_non_cve_version']
                     })
                 else:
-                    script2 = "g.V().has('pecosystem', ecosystem).has('pname', name)" \
-                              ".has('version', version).in('has_version')" \
-                              ".out('has_version').not(out('has_cve')).values('version').dedup();"
```

Comment:
It was added here as a part of the review comment. Instead of doing it every time, we do it only for else case.

Reply:
It is Okay to declare the variable at function start. another option is to put a conditional statement here... which increases cyclometric complexity and hence it holds No value gains at all. 
Declaring `script2` at the start...has No performance loss at all.

Reply:
We dont need any conditions here. script2 variable gets initialized only when there is no value present in the recommended versions. as we are dealing with 0.8 seconds of response time, we dont want to risk any additional logic which is not used later. same is the case with multiple if checks etc. Thats why, im saying create 2 separate function flows. call function x in normal case, function y for snyk case

Reply:
added separate function